conformance/ivm.ion

```diff
@@ -1,10 +1,23 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-(ion_1_x "IVMs don't appear in output data"
-  (toplevel '#$ion_1_0' null '#$ion_1_1' null)
-  (produces null null))
+(document "IVMs don't appear in output data"
+  (each (text      "$ion_1_0")
+        (binary    "E0  01 00   EA")
+        (ivm             1 0)
+        (toplevel '#$ion_1_0')
+        (then (toplevel 1)
+              (each (text      "$ion_1_0")
+                    (binary    "E0  01 00   EA")
+                    (ivm             1 0)
+                    (toplevel '#$ion_1_0')
```

Comment:
Are these synonyms since they are both IVMs that are not tied to a particular encoding? Is there a particular reason you want to test both of them here?

Reply:
I want to test all ways to represent IVMs, to make sure they all work.

This file should probably move into the `core` directory along with other low-level cases that are equally about testing the conformance framework as testing the Ion implementation.